src/gossip/fetcher/fetcher.go

```diff
@@ -132,8 +132,8 @@ func (f *Fetcher) Stop() {
 }
 
 func (f *Fetcher) Overloaded() bool {
-	return len(f.inject) > maxQueuedInjects/2 ||
-		len(f.notify) > maxQueuedAnns/2 ||
+	return len(f.inject) > maxQueuedInjects*3/4 ||
```

Comment:
since maxQueuedInjects is only used here, make maxQueuedInjects = 128 * 3/4 instead.
Same for maxQueuedAnns.

Reply:
It's also used to define the channel